server/views/taskList/taskList.pug

```diff
@@ -28,7 +28,15 @@ block content
         if user.role === 'CA'
             if (['UNSTARTED', 'ELIGIBILITY'].includes(licenceStatus.stage))
                 include eligibilityTask
-                include proposedAddressTask
+
+                if licenceStatus.decisions.eligible
+                    if licenceStatus.tasks.eligibility === 'DONE'
```

Comment:
remove blank line
combine two if statements into one
Looks to me like you can move the task === DONE check into the licenceStatus.isEligible, then this line cleans up to
`if licenceStatus.decisions.eligible`